wgengine/magicsock/debugknobs_stubs.go

```diff
@@ -9,16 +9,18 @@ package magicsock
 
 import "tailscale.com/types/opt"
 
+func ptrTo[T any](v T) *T { return &v }
+
 // All knobs are disabled on iOS and Wasm.
 // Further, they're const, so the toolchain can produce smaller binaries.
```

Comment:
Probably worth removing this line since it's not true any more.

Reply:
Ugh, this changed my mind on the whole plan.

Talking to @maisem here, we decided it'd probably be better to return a `func() T` instead of `*T`. Bonus: holders can't mutate it.

And now the deadcode can still work.

Reply:
Heh, sorry-not-sorry :see_no_evil: